src/routes/index.jsx

```diff
@@ -1,16 +1,32 @@
+
 import React from 'react';
 import { BrowserRouter, Route } from 'react-router-dom';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { faStar, faHeart, faReply } from '@fortawesome/free-solid-svg-icons';
 import Header from '../components/Header';
 import Footer from '../components/Footer';
+// eslint-disable-next-line import/no-named-as-default
 import HomeView from '../containers/HomeView';
 import UserProfile from '../components/user/UserProfile';
+import Login from '../components/Login';
+import RegisterForm from '../components/RegisterForm';
+import GetAnArticle from '../containers/GetAnArticle';
+import '../css/App.css';
+
+
+library.add(faStar);
+library.add(faHeart);
+library.add(faReply);
 
 const Routes = () => (
   <BrowserRouter>
     <div className="App">
       <Header />
       <Route exact path="/" component={HomeView} />
       <Route path="/profile" component={UserProfile} />
+      <Route path="/login" component={Login} />
```

Comment:
Please rebase because we had made it a point to remove the `login` and `register` routes. You will also notice that we don't import `RegisterForm` and `Login` anymore.